src/services/saveImagesToUSB.ts

```diff
@@ -0,0 +1,49 @@
+import { IService } from '../types';
+import { readdirSync, existsSync, mkdirSync, rename, statSync, readdir } from 'fs';
+import { USB_WRITE_PATH } from 'config';
+
+export const SaveImagesToUSB: IService = {
+    execute: async () => {
+        try {
+            //Check if USB is connected
+            const usbConnected = existsSync(USB_WRITE_PATH);
```

Comment:
how often is this called?  should the existence of the USB be a given for this to be called in the first place?  I.e., shouldn't we move that upstream of this function?

Reply:
Agree on this one. 
Though contributor can technically disconnect USB device on-the-go (rare case)
Do you think it's expensive to double-check the existance of this folder before running?